internals/compiler.rst

```diff
@@ -490,7 +490,7 @@ Finally, you need to introduce the use of the new bytecode.  Altering
 places to change. You must add the case for a new opcode into the 'switch'
 statement in the ``stack_effect()`` function in :cpy-file:`Python/compile.c`.
 If the new opcode has a jump target, you will need to update macros and
-'switch' statements in :cpy-file:`Python/peephole.c`.  If it affects a control
+'switch' statements in :cpy-file:`Python/compile.c`.  If it affects a control
```

Comment:
The relevant code was apparently moved into `Python/compile.c` in https://github.com/python/cpython/pull/21517.